src/components/DatePicker/index.android.js

```diff
@@ -1,11 +1,12 @@
 import React from 'react';
 import RNDatePicker from '@react-native-community/datetimepicker';
 import moment from 'moment';
+import _ from 'underscore';
 import TextInput from '../TextInput';
 import CONST from '../../CONST';
 import {propTypes, defaultProps} from './datepickerPropTypes';
 
-class DatePicker extends React.Component {
+class Datepicker extends React.Component {
```

Comment:
Why the name change here? I like the capitalized `P`

Reply:
Ahh just to make it match the names in the other files `index.ios.js` and `index.js`. I can change them all to `DatePicker`.

Reply:
Changed to `DatePicker`

Reply:
rad, thanks!